rollouts/controllers/remoterootsync_controller.go

```diff
@@ -63,7 +63,15 @@ var (
 )
 
 const (
-	externalSyncCreatedConditionType = "ExternalSyncCreated"
+	conditionReady       = "Ready"
+	conditionReconciling = "Reconciling"
```

Comment:
Using both the `Ready` and `Reconciling` conditions seems redundant to me. Are there any situations where the value of the `Ready` condition can not be derived from looking at the `Stalled` and `Reconciling` conditions?

Reply:
Correct, the `Ready` and `Reconciling` conditions are redudant. Originally I thought it was recommended to have the `Ready` condition, but re-reading https://github.com/kubernetes-sigs/cli-utils/tree/master/pkg/kstatus#conditions I see the `Ready` condition is not necessary if the the `Reconciling` and `Stalled` conditions are adopted. I can remove the `Ready` condition.

Reply:
The `Ready` condition is removed in the latest commit.